layouts/layouts.module.css

```
@@ -125,7 +127,9 @@
           lg:max-w-3xl;
 
       > div {
-        @apply w-fit;
+        @apply w-fit
```

```suggestion
        @apply w-full
```

Since the content is wide, using `w-full` instead of `w-fit` seems to make it more responsive at lower resolutions